src/test/java/games/strategy/engine/framework/startup/login/HmacSha512AuthenticatorTest.java

```diff
@@ -155,34 +152,34 @@ public void decodeOptionalProperty_ShouldThrowExceptionWhenValueMalformed() {
     final String name = "name";
     final Map<String, String> properties = ImmutableMap.of(name, "NOT_A_BASE64_VALUE");
 
-    catchException(() -> HmacSha512Authenticator.decodeOptionalProperty(properties, name));
 
-    assertThat(caughtException(), allOf(
-        is(instanceOf(AuthenticationException.class)),
-        hasMessageThat(containsString("malformed")),
-        hasMessageThat(containsString(name))));
+
+    assertThat(assertThrows(AuthenticationException.class,
+        () -> HmacSha512Authenticator.decodeOptionalProperty(properties, name)).getMessage(), allOf(
+            containsString("malformed"),
+            containsString(name)));
   }
 
   @Test
   public void authenticate_ShouldNotThrowExceptionWhenAuthenticationSucceeds() throws Exception {
     final Map<String, String> challenge = HmacSha512Authenticator.newChallenge();
     final Map<String, String> response = HmacSha512Authenticator.newResponse(PASSWORD, challenge);
 
-    catchException(() -> HmacSha512Authenticator.authenticate(PASSWORD, challenge, response));
-
-    assertThat(caughtException(), is(nullValue()));
+    try {
+      HmacSha512Authenticator.authenticate(PASSWORD, challenge, response);
+    } catch (final Exception e) {
+      throw new AssertionError(e);
+    }
```

Comment:
Well, that's the third occurrence. :smile:  I'll leave it up to you if you want to extract an `assertNotThrows()` utility as part of this PR.